config/routes.rb

```diff
@@ -3,6 +3,7 @@
     scope '/:api_key' do
       scope '/content_blocks', controller: :content_blocks do
         post action: :store
+        get action: :batch_fetch
```

Comment:
yes, it's the api part only. the dashboard may use this method - it's only the api part :wink: